hawc/apps/lit/api.py

```diff
@@ -166,6 +169,64 @@ def tag_heatmap(self, request, pk):
         export = FlatExport(df=df, filename=f"df-{instance.id}")
         return Response(export)
 
+    @transaction.atomic()
+    @action(
+        detail=True, methods=("patch",), url_path="replace-hero",
+    )
+    def replace_hero(self, request, pk):
```

Comment:
there need to be some checks here to ensure that the new `hero_ids` area all valid; see `validate_valid_hero_ids`